pandas/tests/io/sas/data/cars.csv

```diff
@@ -0,0 +1,393 @@
+MPG,CYL,ENG,WGT
```

Comment:
Can you make the test data smaller? Shouldn't need close to 400 lines of data IIUC

Reply:
Well, the sas7bdat file is from the test-case from issue, and I can't produce a new one. Also these kind of tests, testing for random bytes in uninitialized memory are somewhat more sensitive with larger test-data. (We are after all dependent on someone writing something nonzero to the memory before we get it, so more is better). 

I suppose I might be able to check that all numbers are integral or close to instead of comparing them to a csv. Would that be preferable?

Reply:
Ok, I decided to check that the floats are integral instead of having a csv fixture, so I resolve this.